include/mockturtle/algorithms/aig_resub.hpp

```diff
@@ -325,6 +325,20 @@ struct aig_resub_functor
         continue;
       }
 
+      if ( true ) // ( ps.fix_bug )
```

Comment:
I wanted to add a parameter to switch between fixed and unfixed, but realized that this is probably specific to AIGs and the `resub_param` is shared among all types of resubstitution.